### ISSUE TITLE
Display Name should be requested if not configured already

### ISSUE THREAD

**Issue by dbareis** [jitsi/jitsi-meet#5775]
**Is your feature request related to a problem you are facing?**
People join the meeting without setting it up.

**Describe the solution you'd like**
If the name is not already set then Jisi should ask for it and save that setting (its a one time request).

**Describe alternatives you've considered**
Asking them to configure it, then I get "it's all too hard"

**Comment by rubenk** [comment]
Did you set requireDisplayName in the config? https://github.com/jitsi/jitsi-meet/blob/master/config.js#L256

**Comment by dbareis** [comment]
I am using "https://meet.jit.si/SomeMeetingRoomName", the point is that people joining the meeting should have set their name but don't.

**Comment by Syonyk** [comment]
This is an option you can configure on your own servers, if you host your own Jitsi instance.  It's not set on the public meet.jit.si server, and I doubt the team will change the configuration away from the normal wide open defaults on that instance.

**Comment by l3d00m** [comment]
I agree that the display name should be prompted at the beginning, we often had the problem that nobody knew who's the "Fellow Jitser" that's now talking. 

But as already pointed out it's possible to configure it for your own deployment and it seems like the Jitsi Team decided not to enable the option on the meet.jit.si deployment. I'll leave it open for a Jitsi Team member to close this if there's no interest in changing this, but I really like this request as well.

**Comment by dbareis** [comment]
I don't have any control over the server for "https://meet.jit.si/".  **It should be the default option**.

**Comment by saghul** [comment]
That's your opinion. I beg to differ. You can choose to enable it on your own deployment. meet.jit.si has it turned off and shall remain that way for the time being.

We are working on some features like a "lobby" where users wait to get let into a meeting, which will require a name to be set.

**Comment by luixxiul** [comment]
FWIW the lobby feature is worked on https://github.com/jitsi/jitsi-meet/pull/6586 etc.

**Comment by lwcorp** [comment]
As this place is for discussing the actual code (which already supports forcing display names), I believe [this meet.jit.si discussion thread](https://community.jitsi.org/t/force-display-name-for-https-meet-jit-si/46980) is a more suited place to continue this.

**Comment by xxjunior** [comment]
The 8x8 asks for the nick name in the start. Is a advantge to force you migrate for the payed service.

**Comment by dbareis** [comment]
Well, I'm using zoom now.

**Comment by saghul** [comment]
🤷‍♀️
